.github/workflows/release.yaml

```diff
@@ -2,8 +2,8 @@ name: Release Charts
 
 on:
   push:
-    branches:
-      - master
+    tags:
+      - "v*"
```

Comment:
Are we going to introduce new release process in our fork?

Reply:
No, it just to have an ability to collect fixes and then have push the button kind of approach